src/translations/en/publication.json

```diff
@@ -131,7 +131,7 @@
     "number_of_pages": "Total number of pages",
     "nvi_header": "Norwegian Science Index - NSI",
     "nvi_success": "This publication can be included in NSI",
-    "nvi_fail_not_rated": "This publication can not be included in NSI, since selected journal is not at level 1 or 2. \n\n(The journal's level may be changed in the Publication Channel Registry, so the publication can ble included in NSI)",
+    "nvi_fail_not_rated": "This publication can not be included in NSI, since selected journal is not at level 1 or 2. \n\n(The journal's level may be changed, so the publication can ble included in NSI)",
```

Comment:
```suggestion
    "nvi_fail_not_rated": "This publication can not be included in NSI, since selected journal is not at level 1 or 2. \n\n(The journal's level may be changed, so the publication can be included in NSI)",
```